scripts/utilities/update_revision.sh

```diff
@@ -514,9 +591,11 @@ for co in "${ATSRC_PACKAGE_CO[@]}"; do
 	if [[ -n "${hash}" ]]; then
 	    echo ""
 	    print_msg 0 "The latest revision of ${repo} is ${hash}"
-	    update_revision ${1} ${hash}
+
+	    version=$(get_version ${package} ${hash})
```

Comment:
What happens if for some reason we reach the `*)` case on `get_version`? If I understood correctly, `$version` here wouldn't be null, but contain something like `"Function get_version doesn't know the package $package"`, which will  trigger some code paths on `send_to_github`.

It would be good to check the return value of `get_version` and only set `$version` here if no error occurred.

Reply:
Done.